docs/manager.md

```diff
@@ -89,18 +89,18 @@ Manager are entitled to collect fees over the investments to insure longetivity
 
 ### Management fee
 
-A fixed 1% management fees has been set in the smart contracts which managers can collect when they move the funds from router to their fund contracts. This amount will be shown on manager dashboard interface. 
+A fixed 1% management fees has been set in the smart contracts which managers can collect when they move the funds from router to their fund contracts. This amount is intended to incentivise the Manager for the costs borne for transaction fees and can be collected from the Manager Dashboard. 
 
 <!-- ### Swap fee
 
 The contract will keep a count of swaps done by the manager and if the fund's performance is above the minimum return set during fund creation they will be allowed to collect the fee they spent on swapping. `Manager is expected to keep the swap fee in base token of the fund, they won't be able to collect the fee in case the funds are invested in other assets` -->
 
 ### Performance fee
 
-As discussed earlier manager will be entitiled to collect the performance fee on meeting the criteria they reach the minimum return they set for themselves during deploying funds. This feature insures managers meet certain targets of profits before collecting any performance fee. 
+As discussed earlier manager will be entitiled to collect the performance fee on each investment upon fulfiling the minimum return they set for their Fund during the creation process. This feature ensures managers meet the predefined returns before collecting any performance fee. 1/10th of the total fee collected by the Manager is allocated to the protocol.
 
 !!! info
-    Manager is expected to keep the performance fee in base token of fund, they wont be able to collect the fee in case the funds are invested in other assets
+    Manager is expected to keep the performance fee amount available in base denomination of the fund when they wish to claim it, they wont be able to collect the fee in case the amount is invested in other assets
```

Comment:
Manager is expected to keep the performance fee amount available in base denomination of the fund when they wish to claim it. They will not be able to collect the fee in the case that the amount is invested in other assets.